core/src/main/res/values/strings.xml

```diff
@@ -196,7 +196,7 @@
     <string name="delete_failed">Unable to delete file. Rebooting the device could help.</string>
     <string name="delete_episode_label">Delete Episode</string>
     <plurals name="deleted_multi_episode_batch_label">
-        <item quantity="one">%d episode selected, %d download deleted.</item>
+        <item quantity="one">One episode selected, %d download deleted.</item>
         <item quantity="other">%d episodes selected, %d download(s) deleted.</item>
```

Comment:
(in reply to your comment on the other string:)
We can only pluralize one number per string. An alternative here would be to omit the word "download" from the second part:
```suggestion
        <item quantity="other">%d episodes selected, %d deleted.</item>
```